### PR TITLE
Norwegian translation of workshopper

### PR DESCRIPTION
**Pull request by leftieFriele**
We're in the process of translating some NodeSchool workshops (https://github.com/nodeschool-no/learnyounode) and we've translated nodeschool.io.

This pr is a continuation of this effort to make more NodeSchool lessons available in Norwegian

### CODE REVIEW

**Comment by marocchino** [comment]
@leftieFriele Thank you for your PR! Do you have any reviewer can confirm this?

File: i18n/usage/nb-no.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  {bold}{green}{appname}{/green} list{/bold}
+    Vis en nylinje-separert liste av alle oppgaver.
+  {bold}{green}{appname}{/green} select NAME{/bold}
+    Velg en oppgave.
+  {bold}{green}{appname}{/green} current{/bold}
+    Vise den gjeldende oppgaven.
+  {bold}{green}{appname}{/green} print{/bold}
+    Skriv ut instruksjonene for gjeldende oppgave.
+  {bold}{green}{appname}{/green} next{/bold}
+    Skriv ut instruksjonene for den neste ikke utførste oppgaven i denne workshoppen.
```
Comment (phillipj) [inline i18n/usage/nb-no.txt]: utførste -> utførte?

File: i18n/nb-no.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      "compare": "Her er den offisielle løsningen i tilfelle du vil sammenligne:\n",
+      "load_error": "FEIL: Klarte ikke skrive ut løsningsfilene: {{{err}}}"
+    }
+  },
+  "ui": {
+    "return": "Skriv '{{appName}}' for å se menyen.\n",
+    "usage": "Bruk: {{appName}} {{mode}} mysubmission.js"
+  },
+  "progress": {
+    "reset": "{{title}} progress reset",
```
Comment (phillipj) [inline i18n/nb-no.json]: Start {{title}} på nytt?

File: i18n/usage/nb-no.txt (hunk 1 above)
Comment (leftieFriele) [inline i18n/usage/nb-no.txt]: Agreed, I rewrote that sentence too as it wasn't really all that great :+1:

File: i18n/nb-no.json (hunk 2 above)
Comment (leftieFriele) [inline i18n/nb-no.json]: :+1:

**Comment by marocchino** [comment]
Is it over? :smile: @leftieFriele @phillipj

**Comment by phillipj** [comment]
Yepp, good one @leftieFriele :+1:

**Comment by marocchino** [comment]
Thank you!

**Comment by leftieFriele** [comment]
When do you plan on making a new workshopper release? I couldn't find any
such info on github.

**Comment by marocchino** [comment]
I don't have permission to do that but, 2.6.0 is out now. :) 
You can see workshopper/organizers#2 for more information.

**Pull request merged**